src/Components/WebAssembly/WebAssembly/src/build/net5.0/Microsoft.AspNetCore.Components.WebAssembly.props

```diff
@@ -0,0 +1,5 @@
+<Project>
```

Comment:
File was deleted during package rename. Unfortunately since template tests are quarantined, we couldn't find this out sooner. Next up, revive template tests.